hms-api/src/repository/domain/Idea.ts

```diff
@@ -33,7 +33,7 @@ class Idea {
   hackathonId: Uuid;
 
   /**
-   *The IDs of the Participants that will work on the Idea
+   *The User IDs of the Participants that will work on the Idea
```

Comment:
maybe point out that the voterIds are also meant to be User IDs 🤔

Reply:
voter and idea participants are meant to be participants and not users my bad